nifi-nar-bundles/nifi-azure-bundle/nifi-azure-nar/pom.xml

```diff
@@ -35,6 +35,12 @@
             <artifactId>nifi-azure-processors</artifactId>
             <version>1.2.0-SNAPSHOT</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-services-api-nar</artifactId>
```

Comment:
Since nifi-azure-processors depends on nifi-standard-processors, this dependency should be changed to nifi-standard-nar, which specifies a dependency on nifi-standard-services-api-nar itself.  This will enable nifi-azure-nar's classloader to have the classes that are depended upon (in nifi-standard-processors) by nifi-azure-processors, which in turn allows nifi-azure-processors to specify nifi-standard-processors as a provided dependency, which fixes the standard processors being loaded twice.

```xml
        <dependency>
            <groupId>org.apache.nifi</groupId>
            <artifactId>nifi-standard-nar</artifactId>
            <type>nar</type>
        </dependency>  
```